src/Toast.tsx

```diff
@@ -28,25 +33,45 @@ const ToastRoot = React.forwardRef((props: ToastProps, ref) => {
   );
 });
 
-type ToastRef = {
-  show: (params: ToastShowParams) => void;
-  hide: (params: ToastHideParams) => void;
+type ToastRefObj = {
+  current: ToastRef | null;
 };
 
-const toastRef = React.createRef<ToastRef>();
+const rootRef: ToastRefObj = {
+  current: null
+};
+const modalRef: ToastRefObj = {
+  current: null
+};
 
 export function Toast(props: ToastProps) {
+  const { setRef } = props;
```

Comment:
you probably would want to do this instead, so `setRef` doesn't also get passed to `ToastRoot` when spreading in all props below.

```
export function Toast({ setRef, ...rest }: ToastProps) {
  return (
    <LoggerProvider enableLogs={false}>
      <ToastRoot ref={setRef} {...rest} />
    </LoggerProvider>
  );
}
```

Reply:
Agreed 👍